### DISCUSSION TITLE
Base Dockerfile

### DISCUSSION THREAD

**Discussion by blacksmithop** [tiangolo/uvicorn-gunicorn-fastapi-docker#335] [Questions] [answered]
Could you please provide the Dockerfile used to create the images you provide?

**Comment by arks-public** [comment]
https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker/tree/master/docker-images

**Comment by blacksmithop** [comment]
@arks-public Thanks, but I was actually curious about the Dockerfile used to create the `tiangolo/uvicorn-gunicorn:python3.xx-xxx` images

**Comment by arks-public** [comment, marked as answer]
@blacksmithop Sounds good, try here: https://github.com/tiangolo/uvicorn-gunicorn-docker/tree/master/docker-images
